<launch>

  <arg name="veh" doc="Name of vehicle. ex: megaman"/>
  <arg name="pkg_name" value="apriltag_ros"/>
  <arg name="node_name" default="apriltag_detector_node"/>
  <arg name="config" default="baseline" doc="Specify a config."/>
  <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman."/>

  <group ns="$(arg veh)">

    <remap from="image_rect" to="camera_node/image/compressed" />
    <remap from="camera_info" to="camera_node/camera_info" />

    <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen">
        <rosparam command="load"
          file="$(find apriltag_ros)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
    </node>

  </group>

</launch>
